src/core/processing/qgsprocessingparameters.h

```diff
@@ -460,6 +461,17 @@ class CORE_EXPORT QgsProcessingParameterDefinition
 
     virtual ~QgsProcessingParameterDefinition() = default;
 
+    /**
+     * A color to represent the default parameter
```

Comment:
```suggestion
     * Returns the color to use for the parameter in model designer windows.
     *
     * The default implementation retrieves the color from the parameter type, see QgsProcessingParameterType::modelColor().
     *
     * \since QGIS 3.44
```